lib/rest-api.php

```diff
@@ -338,6 +338,48 @@ function gutenberg_add_target_schema_to_links( $response, $post, $request ) {
 	return $response;
 }
 
+/**
+ * Include revisions data on post response links.
+ *
+ * @see https://core.trac.wordpress.org/ticket/44321
+ *
+ * @param WP_REST_Response $response WP REST API response of a post.
+ * @param WP_Post          $post The post being returned.
+ * @param WP_REST_Request  $request WP REST API request.
+ * @return WP_REST_Response Response containing the new links.
+ */
+function gutenberg_add_revisions_data_to_links( $response, $post, $request ) {
+
+	$new_links  = array();
+	$orig_links = $response->get_links();
+
+	if ( in_array( $post->post_type, array( 'post', 'page' ), true ) || post_type_supports( $post->post_type, 'revisions' ) ) {
+		// 'version-history' already exists and we don't want to duplicate it.
+		$response->remove_link( 'version-history' );
+
+		$orig_href       = ! empty( $orig_links['self'][0]['href'] ) ? $orig_links['self'][0]['href'] : null;
+		$revisions       = wp_get_post_revisions( $post->ID, array( 'fields' => 'ids' ) );
+		$revisions_count = count( $revisions );
+
+		$new_links['version-history'] = array(
+			'href'  => $orig_href . '/revisions',
+			'count' => $revisions_count,
+		);
+
+		if ( $revisions_count > 0 ) {
+			$last_revision = array_shift( $revisions );
+
+			$new_links['predecessor'] = array(
```

Comment:
Per [this comment](https://core.trac.wordpress.org/ticket/44321#comment:6) and the [referenced RFC](https://tools.ietf.org/html/rfc5829#section-3.5), should this be `predecessor-version` ?